packages/loot-core/migrations/1699454123000_reports.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE reports
+  (id TEXT PRIMARY KEY,
+   name TEXT,
+   start TEXT,
+   end TEXT, 
+   mode TEXT DEFAULT 'total',
+   groupBy TEXT DEFAULT 'Category',
+   balanceType TEXT DEFAULT 'Expense',
+   interval TEXT DEFAULT 'Monthly',
+   empty INTEGER DEFAULT 0,
+   hidden INTEGER DEFAULT 0,
+   uncat INTEGER DEFAULT 0,
+   selectedCategories TEXT,
+   graphType TEXT DEFAULT 'BarGraph',
+   viewLegend INTEGER DEFAULT 0,
+   viewSummary INTEGER DEFAULT 0,
+   viewLabels INTEGER DEFAULT 0,
+   conditions TEXT,
+   conditions_op TEXT DEFAULT 'and',
+   sort_order REAL,
+   data TEXT,
+   colors TEXT,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏color for..? The variable naming could be improved to be more explicit.

Reply:
This is a "future thinking" column. I'd assume at some point we'll add some ability to change color shemes on graphs. This would be where we save the color choice. I'll rename it to be more explicit.